### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by haggishunt56**

### CODE REVIEW

File: apps/docs/src/common/pages/get-involved/working.tsx [hunk 1]
```diff
@@ -56,11 +56,11 @@ const Page: FC<PageProps> = ({ location }) => (
 
           <p>The Home Office design system working group meets once a month to discuss ways to develop and improve the Home Office design system.</p>
 
-          <p>The working group also helps support new design proposals, and encourage the communities to <A href="/getinvolved/propose">get involved and contribute.</A></p>
+          <p>The working group also helps support new design proposals, and encourage the communities to get involved and contribute.</p>
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/get-involved/working.tsx:59]: There is a page for this. I think it's just missing a hyphen but it's worth checking.

File: components/footer/src/Footer.tsx [hunk 2]
```diff
@@ -42,9 +42,9 @@ export const Footer: FC<FooterProps> = ({
               <ul id="navigation" className={classes('navigation-list')}>
                 {navigation.map((v, i) => (
                   <li key={i} className={classes('navigation-item', v.active ? 'active' : undefined)}>
-                    <A classBlock="hods-footer__link" href={v.href} title={v.title}>
+                    <a className="hods-footer__link" href={v.href} title={v.title}>
```
Comment (daniel-ac-martin) [inline components/footer/src/Footer.tsx:45]: This doesn't seem right.

I think it should be:
```jsx
<A classBlock={classes('link')} ...
```

File: docs/working-on-your-project.md [hunk 3]
```diff
@@ -87,9 +87,6 @@ documentation to [Netlify].
 
 [set up your project]: ./get-started
 [pnpm]: https://pnpm.js.org/
-[Applications]: https://not-govuk.netlify.app/
-[Libraries]: https://not-govuk.netlify.app/
-[Components]: https://not-govuk.netlify.app/
```
Comment (daniel-ac-martin) [inline docs/working-on-your-project.md:92]: These could be `https://not-govuk/`.

But it's not such a big deal.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]
Just a few amendments needed.

File: apps/docs/src/common/pages/get-involved/working.tsx (hunk 1 above)
Comment (haggishunt56) [reply apps/docs/src/common/pages/get-involved/working.tsx:59]: closest match is "/get-involved/suggest". Have added this back in.

File: docs/working-on-your-project.md (hunk 3 above)
Comment (haggishunt56) [reply docs/working-on-your-project.md:92]: Have added these back in

File: components/footer/src/Footer.tsx (hunk 2 above)
Comment (haggishunt56) [reply components/footer/src/Footer.tsx:45]: Amended now

**Review by daniel-ac-martin — APPROVED** [review]
I don't think this will have fully fixed the issue (see bottom of: https://611bbe4b63a52334aad2c897--hods.netlify.app/accessibility/links ) but this is definitely an improvement.

**Pull request merged**